scrapers/VirtualRealPorn.yml

```
@@ -15,7 +17,22 @@ movieByURL:
 xPathScrapers:
   sceneScraper:
     scene:
-      Title: //h1[@class="titleVideo"]
+      Title: &titleDef
+        selector: //h1[@class="titleVideo"]/text()
+        postProcess:
+          - replace:
+              - regex:  VR Porn video$
+                with: ""
+              - regex:  VR Trans Porn video$
+                with: ""
+              - regex:  VR Gay Porn video$
+                with: ""
+              - regex:  VR Japanese video$
+                with: ""
+              - regex:  VR Female POV Porn video$
+                with: ""
+              - regex:  VR Amateur Porn video$
+                with: ""
```

```suggestion
          - replace:
              - regex:  VR [\s\w]+? video$
                with: ""
```

Lets simplify this down to a single regex statement. This is also more future proof if they add more sites.

If we wanted to be more precise still we could use something like `VR (?:Trans|Gay|Japanese|Female POV|Amateur)?\s?(?:Porn\s)?video$`